tests/test_api.rs

```diff
@@ -1720,11 +1685,12 @@ fn external_references() {
   let startup_data = {
     let mut snapshot_creator =
       v8::SnapshotCreator::new(Some(&EXTERNAL_REFERENCES));
-    let isolate = snapshot_creator.get_isolate();
-    let mut locker = v8::Locker::new(&isolate);
-    let scope = locker.enter();
     {
-      let mut hs = v8::HandleScope::new(scope);
+      // TODO(ry) this shouldn't be necessary. workaround unfinished business in
```

Comment:
That would be a great area to pay some attention to.

Reply:
Agree. But I want to limit my job here to removing Lockers and adding IsolateHandle (it's already quite a significant patch). This situation with the SnapshotCreator here seems to derive from some bad mutability assumptions from before - namely that v8::Locker was constructed with a `&Isolate` instead of a `&mut Isolate`.